app/src/main/java/com/example/loic/comics_app_android/presentation/ui/activity/ComicListActivity.java

```diff
@@ -0,0 +1,85 @@
+package com.example.loic.comics_app_android.presentation.ui.activity;
+
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.RecyclerView;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+
+import com.example.loic.comics_app_android.R;
+import com.example.loic.comics_app_android.data.model.ResultsItem;
+import com.example.loic.comics_app_android.presentation.ui.adapter.RecyclerComicsListAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ComicListActivity extends AppCompatActivity {
+
+    private RecyclerView mRecyclerView;
+    private RecyclerView.Adapter mAdapter;
+    private RecyclerView.LayoutManager mLayoutManager;
```

Comment:
Je ne pense pas que cette variable soit utile on l'utilise que une seul fois pour set le layoutManager du recycler view je pense qu'il faudrais plutot l'appeler directement dans le recyclerView comme ceci : `mRecyclerView.setLayoutManager(new LinearLayoutManager(this))`

Reply:
✔️